tests/unit/tasks/test_action_subtask.py

```diff
@@ -7,37 +7,35 @@ class TestActionSubtask:
     def test_to_json(self):
         valid_input = (
             "Thought: need to test\n"
-            'Action: {"type": "tool", "name": "test", "activity": "test action", "input": "test input"}\n'
-            "Observation: test observation\n"
+            'Action: {"type": "tool", "name": "test", "path": "test action", "input": "test input"}\n'
+            "<|Response|>: test observation\n"
             "Answer: test output"
         )
 
         task = ToolkitTask(tools=[])
         Pipeline().add_task(task)
         subtask = task.add_subtask(ActionSubtask(valid_input))
-        json_dict = json.loads(subtask.action_to_json())
+        json_dict = json.loads(subtask.request_to_json())
 
-        assert json_dict["type"] == "tool"
         assert json_dict["name"] == "test"
-        assert json_dict["activity"] == "test action"
+        assert json_dict["path"] == "test action"
```

Comment:
API language

Reply:
Sticking with "path" for now.

Reply:
Should we rename the activity mixin then? Might be kind of confusing for developers that Activity -> Path.